### DISCUSSION TITLE
How-To submit an issue/help request with updating MS Learning Pathways

### DISCUSSION THREAD

**Discussion by jamesshover** [community/community#140794] [New to GitHub]
Downloaded V5 MS Learning Pathways and updated in Tenant. Went to learning Pathways SPO site > Site Contents > MS Learning Pathways and requested update which updated.
Went back to site and selected link and it shows no data.
Went to Custom Config list, list view, and there are two Custom Config items both with creation date 2/11/202 and both show modification 42 minutes ago (when I updated the file).
When I select M365 training I receive an error "Microsoft 365 learning pathways has a configuration issue."

**Comment by ghostinhershell** [comment]
Hi @jamesshover, thanks so much for reaching out in our community! 🎉 

Microsoft 365 learning pathways is an open source project supported through our [online issues list](https://aka.ms/CustomLearningHelp) on GitHub. Note that the learning pathways solution and its components aren't covered by any existing Microsoft support contract.

**Reply by jamesshover** [reply]
Good afternoon,
Where can I get assistance with this issue? The learning Pathways site is no longer working and all links to it are not working. What is the best way to get this issue resolved?

Note typo-Creation date of custom config file was 2/11/2020
Best, JS
